### ISSUE TITLE
Invalid YAML is generated when exporting a deployment

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#2486]
In a rel-11_0 installation I exported a simple deployment. The exported file claims to be a YAML file. However `yamllint` has a different opinion.

```
bernhard@bernhard-Aspire-A515-57:~/devel/OTOBO/otobo$ cat ../ImportExportFolder/Export_Deployment_8.yml 
---
Modified:
  DynamicField::Namespaces:
    ChangeTime: 2023-08-22 11:08:53
    CreateTime: 2023-08-22 11:08:53
    DefaultID: 356
    DefaultVersionID: 356
    DeploymentTimeStamp: 2023-08-22 11:08:53
    EffectiveValue:
    - Computer
    - Hardware
    - Location
    - Network
    - Software
    - NamespaceBoden
    IsValid: 1
    ModifiedID: 7
    ModifiedVersionID: 9
    Name: DynamicField::Namespaces
    ResetToDefault: 0
    TargetUserID: ~
    UserModificationActive: 0
bernhard@bernhard-Aspire-A515-57:~/devel/OTOBO/otobo$ yamllint ../ImportExportFolder/Export_Deployment_8.yml 
../ImportExportFolder/Export_Deployment_8.yml
  10:5      error    wrong indentation: expected 6 but found 4  (indentation)


```

**Comment by bschmalhofer** [comment]
Looks like I was fooled by `yamllint` here. The program complained about missing indention for the list in:

```
    EffectiveValue:
    - Computer
    - Hardware
    - Location
    - Network
    - Software
    - NamespaceBoden
    IsValid: 1
```
But the YAML spec says that the item marker `-` already counts as indention, so the snippet is valid YAML. 

I could not immediately find a YAML validator, so I used `yq -o json Export_Deployment_8.yml` for double checking. That command transformed YAML into JSON without complaints. So the generated file semms to be valid YAML. Double-doublechecking I found that breaking the YAML file also broke the transformation. This is a good thing.
